core/scripts/umip-3/Propose.js

```diff
@@ -73,7 +73,10 @@ async function runExport() {
 
   console.log("Deploying new Store contract.");
 
-  const store = await Store.new(zeroAddress, { from: proposerWallet });
+  const regularFee = { rawValue: web3.utils.toWei("0.2", "ether") };
```

Comment:
The `Propose` script was merged before PR [1256](https://github.com/UMAprotocol/protocol/pull/1256)